javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,102 @@
+console.log('JS1 WEEK4');
+
+const listOftodos = []
+const storeName = []
+const calcSentence = []
+function getReply(command) {
+
+   if (command.includes('Hello my name is') || command.includes('my name is')){
+    const splitNameSentence = command.split(' ')
+    const rememberName = splitNameSentence.length <= 4 ? splitNameSentence[3] : splitNameSentence[4];
+    storeName.push(rememberName)
+    console.log('Nice to meet you ' + rememberName);
+   } 
+
+   if (command === 'What is my name' && storeName.length >= 1){
+    const youName = 'Your name is '
+    console.log( youName + storeName[0]);
+   } else if (storeName.length == 0){
+       console.log('Your name is not mentioned yet');
+   }
+
+
+   if (command.includes('add fishing to my todo')){
+       listOftodos.unshift('fishing')
+       console.log('fishing added to your todo list ');
+   }
+
+
+   if (command.includes('add singing in the shower to my todo')){
+    listOftodos.push('singing in the shower')
+    console.log('Singing in the shower added to your todo list ');
+}
```

Comment:
Good. And maybe there's also a shorter way to write this logic? Maybe you can pass every todo as a ${newTodo} ?